pkgs/development/python-modules/nbconflux/default.nix

```diff
@@ -2,26 +2,31 @@
   lib,
   buildPythonPackage,
   fetchFromGitHub,
-  isPy27,
   nbconvert,
   pytestCheckHook,
   requests,
   responses,
+  setuptools,
+  versioneer,
 }:
 
 buildPythonPackage rec {
   pname = "nbconflux";
   version = "0.7.0";
   format = "setuptools";
```

Comment:
Since we’re using `build-system` definitions now, this should be the modern `pyproject = true;` to use the new Python build system, and `propagatedBuildInputs` should be renamed to `dependencies`.

Reply:
Done. Will push momentarily.